### ISSUE TITLE
Setting Default Values. e.g. vehicle type. 

### ISSUE THREAD

**Issue by akobelan** [gis-ops/valhalla-app#212]
### Description

It would be nice if the vehicle type could be set in a config file so that it is automatically selected when the program starts up.

**Comment by chrstnbwnkl** [comment]
Yeah that would be a good addition indeed. Could be added as an environment variable. Do you want to give it a shot?
